requirements.txt

```
@@ -2,3 +2,4 @@ Flask==2.0.3
 karton-core>=4.2.0,<5.0.0
```

```suggestion
karton-core>=4.4.0,<5.0.0
```

We need a version bump to have access to `backend.get_outputs`